recipes/callerpp/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -x -e -o pipefail
+
+make \
+  CC="${CC}" \
```

Comment:
You will need to pass also `CPP=${CXX}` (https://github.com/nh13/callerpp/blob/8e1b1ee1d350880891f765aadc7ab0bb28c027db/Makefile#L4)
The convention is to use `CXX` for c++ compiler, and `CPP` for the C pre-processor. Since you are the author of callerpp I'd recommend to replace CPP with CXX at https://github.com/nh13/callerpp/blob/8e1b1ee1d350880891f765aadc7ab0bb28c027db/Makefile#L4

Reply:
https://github.com/nh13/callerpp/pull/12